app/models/observation.rb

```diff
@@ -64,16 +64,32 @@ class Observation < ApplicationRecord
 
   has_rich_text :description
 
-  before_save :add_points_for_interventions
+  before_save :add_points_for_interventions, if: :intervention?
+  before_save :add_points_for_included_images, if: proc { |observation| observation.activity? || observation.intervention? }
 
-private
+  private
 
-  def add_points_for_interventions
+  def add_points_for_included_images
```

Comment:
This isn't quite right. If a school records an activity or intervention for a previous school year and adds a photo, they'll still get the bonus points for the image. They should only get the bonus if they're actually scoring points. 

So think it needs a check to confirm that the bonus is only added if the score is non zero?